config/routes.rb

```diff
@@ -32,6 +32,7 @@
   resources :posts, only: %i(show destroy) do
     member do
       get :interactions
+      get :mentionable_in_comment
```

Comment:
Do I rename `PostService#mentionable_in_comment` also? I think we should leave it, since `PostService#mentionable` would be ambigous.